tests/framework/microvm.py

```diff
@@ -300,6 +312,7 @@ def setup(self):
     def spawn(self, create_logger=True):
         """Start a microVM as a daemon or in a screen session."""
         # pylint: disable=subprocess-run-check
+        # pylint: disable=too-many-statements
```

Comment:
I think one way to avoid this would be to move, for example, the following part:
```python
self.actions = Actions(self._api_socket, self._api_session)
...      
self.vsock = Vsock(self._api_socket, self._api_session)
```
to a separate function (`initialize_resources()` or something like that).

`spawn()` is pretty huge at this moment.

Reply:
done